libs/SalesforceSDK/src/com/salesforce/androidsdk/ui/components/PickerBottomSheet.kt

```diff
@@ -0,0 +1,631 @@
+                                                if (listItem is UserAccount) {
+                                                    UserAccountListItem(
+                                                        displayName = listItem.displayName,
+                                                        loginServer = listItem.loginServer,
+                                                        selected = selected,
+                                                        onItemSelected = { onItemSelected(listItem, true) },
+                                                        profilePhoto = listItem.profilePhoto?.let {
+                                                            painterResource(
+                                                                it.generationId
+                                                            )
+                                                        },
+                                                    )
+                                                /*
+                                                 TODO: Remove this mock when a UserAccount can be created in without
+                                                 SalesforceSDKManger (for previews).  This would be trivial with an
+                                                 internal constructor if the class was converted to Kotlin.
+                                                */
+                                                } else if (listItem is UserAccountMock) {
+                                                    UserAccountListItem(
+                                                        displayName = listItem.displayName,
+                                                        loginServer = listItem.loginServer,
+                                                        selected = selected,
+                                                        onItemSelected = { onItemSelected(listItem, true) },
+                                                        profilePhoto = listItem.profilePhoto?.let {
+                                                            painterResource(
+                                                                it.generationId
+                                                            )
+                                                        },
+                                                    )
+                                                }
+                                            }
+                                        }
+                                    }
+                                    HorizontalDivider(
+                                        thickness = STROKE_WIDTH.dp,
+                                        modifier = Modifier.padding(horizontal = PADDING_SIZE.dp),
+                                        color = colorScheme.surfaceVariant,
+                                    )
+
+                                    // Add New Connection/Account Button
```

Comment:
Adding this to the end of the list feels a bit obtuse, but having the add button as the last list item really helps to make the UI more usable on a phone in landscape when the list is very cramped.